<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Research Areas</title>

<script src="site_libs/header-attrs-2.21/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.4.0/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.0/css/v4-shims.min.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Begonia Society</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
<li>
  <a href="research.html">
    <span class="fa fa-bug"></span>
     
    Research
  </a>
</li>
<li>
  <a href="art.html">
    <span class="fa fa-moon-o"></span>
     
    Art
  </a>
</li>
<li>
  <a href="music.html">
    <span class="fa fa-music"></span>
     
    Music
  </a>
</li>
<li>
  <a href="http://news.begoniasociety.org/">
    <span class="fa fa-rss"></span>
     
    Blog
  </a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="mailto:moon@begoniasociety.org">
    <span class="fa fa-envelope"></span>
     
    Contact me
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Research Areas</h1>

</div>


<p><img align="right" src="https://seltmann.github.io/begoniasociety/images/Rplot01.png" width="300">
My principal research interests are to digitally capture biodiversity
information using materials from natural history collections and to
employ these data to analyze, model, and decode patterns of
environmental, morphological, and community change. The patterns provide
insights into the classification of organisms, the evolution of
phenotype, and the ecological function of traits. Each specimen is an
object that contains many associated pieces of information. This
information includes the occurrence of the specimen in space, time, or
other parameters, its internal and external anatomy, DNA, stable isotope
and chemical signatures, associated publications, images, and the
individuals who studied or collected the organisms. Natural history
collections comprise libraries of millions of biological specimens,
ranging from preserved pollinating bees to dinosaur remnants. These
collections can shed light on environmental change and morphological
evolution over long periods, illuminate how species are lost, and how
biological knowledge has evolved. As a leader and key contributor to
significant specimen digitization and data aggregation efforts, I have
helped initiate, produce, and advance important regional and global
datasets. My research efforts are also guided by the recognition that
collections serve as invaluable hubs for research on specific taxa and
are integral to our global cultural heritage.</p>
<p>Much of my research addresses fundamental questions in Hymenoptera
systematics and biodiversity data sharing. However, my research also
addresses interactions and communities of organisms and thus includes
investigations involving other insects and kingdoms, including plants,
birds, and reptiles. Currently, we are investigating trait evolution
through new computational tools, new methods for sharing biodiversity
data, and new efforts for describing insect-plant interactions and
distributions for conservation.
<img align="left" src="https://seltmann.github.io/begoniasociety/research_files/figure-html/globi-1.png" width="400"></p>
<div id="biocollections" class="section level2">
<h2>Biocollections</h2>
<p>Central to my research are natural history collections - massive sets
of biological specimens that are housed in museums, tissue collections,
herbaria, botanical gardens, and zoos, and that have been compiled over
several centuries. Together, they constitute our biodiversity libraries.
These collections have broad cultural significance, and more
importantly, constitute enormous primary resources that support
fundamental research in the biological sciences. These libraries of
millions of biological specimens, ranging from preserved pollinating
bees to dinosaur remnants, shed light on the past, and illuminate ways
that the climate has changed, that species are lost, and how biological
knowledge has evolved. They also constitute voucher repositories for
specimens used in research and identification of organisms of all
varieties.</p>
</div>
<div
id="investigating-traits-and-morphology-via-biodiversity-data-science-and-computer-vision"
class="section level2">
<h2>Investigating Traits and Morphology via Biodiversity Data Science
and Computer Vision</h2>
<p>A key research emphasis in my lab is to develop and apply new
computational imaging and data science tools for fundamental
investigations in morphology, anatomy, and phenotype. This research area
is exemplified by a major NSF initiative I am leading, <i>Extending
Anthophila research through image and trait digitization
(Big-Bee)</i><sup>1</sup>. A cornerstone of this 13-institution effort
is the high-resolution imaging of more than one million specimens,
representing more than 5000 bee species, in 2D and 3D modalities. These
data and novel computational tools we are developing in the project will
enable quantitative analyses of morphological features and will furnish
new automated tools that aid identification. Recent advancements in
computer vision and machine learning are transforming many fields of
science, including entomology<sup>2</sup>. These emerging computational
methods are already yielding striking results, including impressive
capacities for sorting organisms<sup>3</sup> and quantifying
abundances.</p>
<p>My research group is investigating multiple computer vision
techniques (image segmentation, feature identification) and machine
learning methods (data augmentation, transfer learning) for the
quantification of bee traits, the analysis of variation between species
or populations, and the development of new morphological characters for
identification and systematic study. A simple but highly effective
example from our current work involves the computational quantification
of bee pilosity (Fig 1), a trait that may affect thermal tolerance and
the ability of a species to adapt to changing climate conditions. Bee
hairs have also evolved for carrying pollen, and clades differ markedly
in pilosity patterns. However, hair patterns, like many morphological
structures, are variable and difficult to quantify in a repeatable and
scalable manner and thus exemplify the potential value that
computational tools for trait analysis may contribute.</p>
<p>In addition to investigating new character systems, my research group
is developing scalable, geographic-specific specimen classifiers using
training data from images captured from natural history collections. The
overarching vision of this effort is to develop methods for creating
individualized models that can be used as “keys” that can be trained to
encode expert domain knowledge of taxonomists or researchers.</p>
<p><img align="center" src="https://seltmann.github.io/begoniasociety/images/segmentation.jpg"></p>
</div>
<div id="advancing-biodiversity-data-and-knowledge-sharing"
class="section level2">
<h2>Advancing Biodiversity Data and Knowledge Sharing</h2>
<p>Access to biological information is dramatically reshaping the kinds
of research questions we can address and raising questions about how we
generate and manage biological data. Through my research, I examine
processes through which we capture information for research in digital
formats, use and reuse large datasets, publish and share biological
data, preserve at-risk information, and use this information in
scientific activities. My work is consequently aligned with major
research initiatives in biodiversity that seek to enable researchers to
access and exploit biodiversity data troves through global computing
systems and new technologies. Presently, I am working with a global
network of collaborators on the Extended Specimen Concept<sup>4</sup>, a
now widely accepted concept that I co-authored for the future of data
sharing. This initiative is working to promote and interconnect emerging
multimodal specimen datasets, including 3D image models<sup>5</sup>,
environmental data, and trait data (as in the Global Bee Interaction
Database<sup>6</sup>). One example of my research activity in this
overarching area is the publication “A semantically enriched taxonomic
revision of <i>Gryonoides</i> Dodd, 1920 (Hymenoptera, Scelionidae),
with a review of the hosts of Teleasinae.” (J. Hymenoptera Res., 2021).
This work exploits data sharing of examined material and translation
into PhenoScript, a standardized language for linking morphology with
ontology to explicitly describe characters.
<img align="left" src="https://seltmann.github.io/begoniasociety/images/extended-specimen.jpg" width="300"></p>
<p>Biological knowledge has expanded greatly, but the reusability of
prior findings has been historically limited by the unstructured way in
which primary contributions, such as species descriptions, have been
documented and published. By structuring scientific language and
modeling the knowledge it embodies, we can greatly enhance the utility
of current and past research literature for deeper meta-analyses. I
approach this topic as a problem involving the development of structured
language in the form of domain-specific ontologies. When designed
appropriately, such an ontology can embody trophic-level network
assemblages, supporting the computational discovery of biological
relations and providing ecological links to gene annotation efforts, and
can even facilitate knowledge discovery from prior ecological
literature. To realize this vision, I am developing an ontology for bee
functional traits that may revolutionize how we describe and monitor
bees and enable researchers to harvest this information in biodiversity
informatics studies. This ontology will align with the already existing
insect morphological ontologies and will also yield a novel
repository.</p>
</div>
<div id="insect-distributions-and-conservation" class="section level2">
<h2>Insect Distributions and Conservation</h2>
<p>A further research focus in my lab, building on my prior work, will
be aimed at understanding and mitigating insect biodiversity loss.
Understanding such declines requires characterizing the species
distributions of insects and their host plants. My group investigates
these questions by analyzing camera trap data and through biodiversity
data-driven analyses of distributions. We complement these methods with
basic fieldwork to enable us to evaluate hypotheses via ground-truth
data. Presently, my collaborators and I are developing new techniques
for analyzing co-occurrences of plants and bees from collection data,
which make it possible to identify shifts or phenological mismatches.
Our ongoing work includes taxonomic evaluations of bees on Santa Cruz
Island, California, to produce a checklist for this scientifically
valuable island ecosystem<sup>7</sup>. Species checklists are a key
product of systematic research and also furnish valuable resources for
informing predictions of multi-species occupancy models, ecological
networks, and other community-level classifications. I am complementing
these activities by developing new computer vision approaches for
automating the identification and prediction of new species (Fig. 3).
The methods we are developing for research can be applied to important
conservation challenges, and our research engages state and federal
agencies, including USFW and USGS. We have applied these methods in
recent characterizations of the insects associated with the endangered
California lupine plant<sup>8</sup> and in a genome skimming analysis of
herbarium specimens aimed at characterizing populations of an estuarine
seablite (Madroño Journal, Accepted, 2023).
<img align="right" src="https://seltmann.github.io/begoniasociety/images/SCI.jpg" width="300"></p>
</div>
<div id="references" class="section level2">
<h2>References</h2>
<ol style="list-style-type: decimal">
<li>Katja C. Seltmann (2021). Extending Anthophila research through
image and trait digitization (Big-Bee) proposal. UC Santa Barbara:
Cheadle Center for Biodiversity and Ecological Restoration. Retrieved
from <a href="https://escholarship.org/uc/item/2vm761mv"
class="uri">https://escholarship.org/uc/item/2vm761mv</a></li>
<li>Høye, Toke T., Johanna Ärje, Kim Bjerge, Oskar LP Hansen, Alexandros
Iosifidis, Florian Leese, Hjalte MR Mann, Kristian Meissner, Claus
Melvad, and Jenni Raitoharju. “Deep learning and computer vision will
transform entomology.” Proceedings of the National Academy of Sciences
118, no. 2 (2021): e2002545117.</li>
<li>Spiesman, Brian J., Claudio Gratton, Richard G. Hatfield, William H.
Hsu, Sarina Jepsen, Brian McCornack, Krushi Patel, and Guanghui Wang.
“Assessing the potential for deep learning and computer vision to
identify bumble bee species from images.” Scientific reports 11, no. 1
(2021): 1-10.</li>
<li>James Lendemer, Barbara Thiers, Anna K Monfils, Jennifer Zaspel,
Elizabeth R Ellwood, Andrew Bentley, Katherine LeVan, John Bates, David
Jennings, Dori Contreras, Laura Lagomarsino, Paula Mabee, Linda S Ford,
Robert Guralnick, Robert E Gropp, Marcy Revelez, Neil Cobb, Katja
Seltmann, M Catherine 5. Aime, The Extended Specimen Network: A Strategy
to Enhance US Biodiversity Collections, Promote Research and Education,
BioScience, Volume 70, Issue 1, January 2020, Pages 23–30, <a
href="https://doi.org/10.1093/biosci/biz140"
class="uri">https://doi.org/10.1093/biosci/biz140</a></li>
<li>Katja C. Seltmann, Jorrit H. Poelen, UCSB Natural History
Collections, &amp; Big Bee Community. (2023). Halictus ligatus
Halictidae UCSB-IZC00036803 [Data set]. Zenodo. <a
href="https://doi.org/10.5281/zenodo.7510912"
class="uri">https://doi.org/10.5281/zenodo.7510912</a></li>
<li>Katja C. Seltmann. (2022). Global Bee Interaction Data (v2.02) [Data
set]. Zenodo. <a href="https://doi.org/10.5281/zenodo.7315159"
class="uri">https://doi.org/10.5281/zenodo.7315159</a></li>
<li>Katja C. Seltmann, David Dewey, Lynn McLaren, &amp; Charlie Thrift.
(2022). Native and non-native bees (Anthophila) of Santa Cruz Island: An
annotated checklist (0.03). Zenodo. <a
href="https://doi.org/10.5281/zenodo.7216845"
class="uri">https://doi.org/10.5281/zenodo.7216845</a></li>
<li>C. I. Motta, J. C. Luong, K. C. Seltmann. Plant–arthropod
interactions of an endangered California lupine. Ecology and Evolution.
2022 Mar;12(3):e8688.</li>
</ol>
<center>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
